atomic_reactor/plugin.py

```diff
@@ -210,6 +214,9 @@ def __init__(self, dt, workflow, plugin_class_name, plugins_conf, *args, **kwarg
         self.workflow = workflow
         super(BuildPluginsRunner, self).__init__(plugin_class_name, plugins_conf, *args, **kwargs)
 
+    def on_plugin_failed(self):
+        self.workflow.build_has_failed = True
```

Comment:
Let me explain another way. When we set up automated rebuilds, we only want to rebuild from Builds (OpenShift Build objects) that built successfully in the first place.

That's what this instance variable is intended to be: will this Build get marked by OpenShift as completed or failed?